graphs/util/db.py

```diff
@@ -4742,3 +4744,112 @@ def constructResponse(statusCode, message, error):
 		response['Error'] = error
 
 	return response
+
+
+def add_share_graph_event(graph_id, owner_id, group_id, member_id):
+	'''
+		Add a new share graph event to the table.
+		After sharing the graph with a group this function will create
+		a share graph event for all the users in that group
+
+		@param graph_id: id of the graph shared
+		@param owner_id: owner of the graph which is shared
+		@param group_id: id of the grop
+		@param member_id: id of the member the graph is shared
+	'''
+	# Create database connection
+	db_session = data_connection.new_session()
+
+	# Get the current time
+	cur_time = datetime.now()
```

Comment:
we dont need this variable. you directly assign the value while creating the share event object.